src/searchee.ts

```diff
@@ -87,10 +88,17 @@ export async function createSearcheeFromPath(
 		(runningTotal, file) => runningTotal + file.length,
 		0,
 	);
+	const seasonMatch =
+		basename(filepath).length < 12
+			? basename(filepath).match(SONARR_SUBFOLDERS_REGEX)
+			: undefined;
```

Comment:
Why do we need the length check? The regex `/^(?:S(?:eason )?(?<season>\d{1,4}))$/i` already implicitly limits the size to `Season 1234` at the maximum due to `^` and `$`.

Reply:
I was trying to speed up the exclusion process, as regex steps take more cycles than simple length checking.